lib/network/modules/Clustering.js

```diff
@@ -907,7 +906,7 @@ class ClusterEngine {
   /**
   * Get a stack of clusterEdgeId's (+base edgeid) that a base edge is the same as. cluster edge C -> cluster edge B -> cluster edge A -> base edge(edgeId)
   * @param {vis.Edge.id} edgeId
-  * @returns {Array<vis.Edge.id>}
+  * @returns {Array.<vis.Edge.id>}
```

Comment:
? `id` is not a type, is it???

Reply:
No, I'm going to go back to change all of these to a type def'd id.

The namespacing of `vis.` is also incorrect.

Reply:
OK. In next iteration I presume?

Reply:
Yes